daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ExceptionTest.scala

```diff
@@ -533,21 +539,70 @@ class ExceptionTest extends AnyWordSpec with Inside with Matchers with TableDriv
       val transactionSeed: crypto.Hash = crypto.Hash.hashPrivateKey("transactionSeed")
 
       val testCases = Table[String, Boolean](
-        ("update", "caught"),
-        ("exercise @M:T BodyCrash cid ()", true),
-        ("exercise @M:T ControllersCrash cid ()", false),
-        ("exercise @M:T ObserversCrash cid ()", false),
-        ("exercise_interface @M:I BodyCrash (COERCE_CONTRACT_ID @M:T @M:I cid) ()", true),
-        ("exercise_interface @M:I ControllersCrash (COERCE_CONTRACT_ID @M:T @M:I cid) ()", false),
-        ("exercise_interface @M:I ObserversCrash (COERCE_CONTRACT_ID @M:T @M:I cid) ()", false),
+        ("udpate", "caught"),
+        """
```

Comment:
Do we need any additional testing around views failing?

Reply:
I will consider the proposal of Moritz.